scipy/io/_fast_matrix_market/__init__.py

```diff
@@ -13,7 +13,7 @@
 import os
 
 import numpy as np
-import scipy.sparse
+import scipy.sparse as SP
```

Comment:
I'm not sure that I like this alias. I'd rather we spell out the names, or import things directly.